pep/app/pods/application/adapter.ts

```diff
@@ -66,10 +76,12 @@ export default class Application extends DS.RESTAdapter.extend(DataAdapterMixin,
      * @returns Object
      */
     get headers() {
-        const headers = { client_id: ENV.clientId } as any;
+        const headers = { 'client-id': ENV.clientId } as any;
         if (this.session.isAuthenticated) {
-            const { access_token } = this.session.data!.authenticated;
-            headers['Authorization'] = `Bearer ${access_token}`;
+            // We are converting to unknown because session data is specified as something
+            // completely different by the addon
```

Comment:
we've custom defined this `authenticated ` session data interface ourselves, in `types/ember-simple-auth/services/session.d.ts`. So its probably better to just change it there directly, to `PepSecureAuthenticatedData` so we don't need to convert here or anywhere else.

Reply:
I'd tend to disagree with this on the basis that eventually we won't be specifying those types as they will provide them and in that case we will have to do this anyways

Reply:
@bakerac4 totally agree on that point (that we can remove all ember-simple-auth specific types) once they have types defined themselves. 

But in this case, our `authenticated` interface is custom and API-dependent (the existing `SessionAuthenticatedData` interface we defined is for OAuth2, so wouldn't even apply here even if ESA had types for oauth2 structures, since we made our own custom authenticator). So i still think that if/when we get types from ESA, we would and should be still be directly extending/typing that one interface at the module level to be what really is, instead of it "starting" as oauth2.